packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/drp/configuration/common/mainPcc/dedicatedCloud-datacenter-drp-mainPccStep.component.js

```diff
@@ -9,7 +9,7 @@ export default {
     datacenters: '<datacenterList',
     defaultLocalVraNetwork: '<?',
     drpInformations: '=',
-    getIpOrderLink: '<',
+    getStateToOrderIp: '<',
```

Comment:
```suggestion
    orderIpLink: '<',
```
? 
Component might not have to know whether the link is generated from a state or not

Reply:
Good idea